### PR TITLE
Band crud

### PR DESCRIPTION
**Pull request by DankaMarci**
Migrated the prisma schema to add a default value to the bandmembership's status attribute.

### CODE REVIEW

File: apps/backend/src/band/band.service.ts [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  async findMembers(id: number): Promise<User[]> {
+    try {
+      const bandmemberships = await this.prisma.bandMembership.findMany({ where: { bandId: id } });
+      if (!bandmemberships) throw new Error();
+      const members = await this.prisma.user.findMany({
+        where: { id: { in: bandmemberships.map((membership) => membership.userId) } },
+      });
+      return members;
+    } catch (error) {
+      throw new NotFoundException('No members found');
```
Comment (justnyx) [inline apps/backend/src/band/band.service.ts:67]: If a band has no members, shouldn't it be deleted? If so, how should this be implemented? @balintking

File: apps/backend/src/band/band.service.ts [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+      if (!res) throw new Error();
+      return res;
+    } catch (error) {
+      throw new NotFoundException('No bands found');
+    }
+  }
+
+  async findMembers(id: number): Promise<User[]> {
+    try {
+      const bandmemberships = await this.prisma.bandMembership.findMany({ where: { bandId: id } });
```
Comment (justnyx) [inline apps/backend/src/band/band.service.ts:60]: Only those with `ACCEPTED` membership status should be returned in my opinion.

**Review by justnyx — CHANGES_REQUESTED** [review]
I've left some comments, please fix where necessary.

**Review by balintking — CHANGES_REQUESTED** [review]
Remove manual Error throws

**Review by FearsomeRover — APPROVED** [review]

**Review by balintking — APPROVED** [review]

**Pull request merged**